tracer/src/Datadog.Trace/Tagging/CosmosDbTags.cs

```diff
@@ -17,6 +17,9 @@ internal partial class CosmosDbTags : InstrumentationTags
         [Tag(Trace.Tags.InstrumentationName)]
         public string InstrumentationName => nameof(IntegrationId.CosmosDb);
 
+        [Metric(Trace.Tags.Measured)]
+        public double? Measured => 1.0;
```

Comment:
We do have Cosmosdb tests, the simulator is just so flaky  we don't run them as standard. You may want to do a run to update the snapshots